scania-digital-design-system/sdds
# feat: chips & message components

**Describe pull-request**  
Two new components. Chips and Messages

**Solving issue**  
Fixes: [AB#1730](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1730) and [AB#1904](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/1904)

**How to test**  
1. Go to the storybook
2. check chips, go thru the storie and see if they are correct
3. check messages, check the stories, and use the controls to se if you can create all variants.
